src/Microsoft.ML.EntryPoints/TrainTestMacro.cs

```diff
@@ -84,10 +82,6 @@ public sealed class Output
                 "provided as the Input.TransformModel.", SortOrder = 1)]
             public PredictorModel PredictorModel;
 
-            [TlcModule.Output(Desc = "The final model including the trained predictor model and the model from the transforms, " +
-                "provided as the Input.TransformModel.", SortOrder = 2)]
-            public TransformModel TransformModel;
```

Comment:
this is not needed anymore? I believe we added it because there was no other way to save the final transforms model by itself.

Reply:
It was only used in the legacy Pipeline API scenario, since the models created by the pipelines were TransformModels (they included the scorer), so they are not needed anymore.

---
In reply to: [242420400](https://github.com/dotnet/machinelearning/pull/1898#discussion_r242420400) [](ancestors = 242420400)

Reply:
Very interesting! The sort of ridiculous web you had to weave to get this API to do anything with itself was quite something. 😄